frontend/frontend.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Install dependencies
+apt-get update && apt-get install -y libglib2.0-dev libfdt-dev libpixman-1-dev zlib1g-dev ninja-build libcap-ng-dev libslirp-dev
+
+
+# Install qemu
```

Comment:
please use a variable to represent the version number in order to maintain when we use to change the version, we just to change the variable.